### PR TITLE
Test against Docker 1.6.2 and 1.7.1

### PR DESCRIPTION
**Pull request by aanand**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -48,16 +48,16 @@ RUN set -ex; \
...
+ENV ALL_DOCKER_VERSIONS 1.6.2 1.7.1
 
 RUN set -ex; \
-    curl https://get.docker.com/builds/Linux/x86_64/docker-1.6.0 -o /usr/local/bin/docker-1.6.0; \
-    chmod +x /usr/local/bin/docker-1.6.0; \
-    curl https://test.docker.com/builds/Linux/x86_64/docker-1.7.0 -o /usr/local/bin/docker-1.7.0; \
-    chmod +x /usr/local/bin/docker-1.7.0
+    curl https://get.docker.com/builds/Linux/x86_64/docker-1.6.2 -o /usr/local/bin/docker-1.6.2; \
+    chmod +x /usr/local/bin/docker-1.6.2; \
+    curl https://test.docker.com/builds/Linux/x86_64/docker-1.7.1 -o /usr/local/bin/docker-1.7.1; \
```
Comment (thaJeztah) [inline Dockerfile]: s/test.docker.com/get.docker.com/

Comment (aanand) [inline Dockerfile]: Updated.

**Comment by thaJeztah** [comment]
LGTM (fwiw), but there's a failing test;

```
ERROR: test_env_file_relative_to_compose_file (tests.integration.cli_test.CLITestCase)
```

**Comment by aanand** [comment]
It's that transient Docker 1.6 race condition. I've given it a kick.

**Pull request merged**